src/base/http/types.h

```diff
@@ -70,6 +70,8 @@ namespace Http
     const char CONTENT_TYPE_FORM_ENCODED[] = "application/x-www-form-urlencoded";
     const char CONTENT_TYPE_FORM_DATA[] = "multipart/form-data";
 
+    const char COOKIE_SESSION_ID[] = "SID";
```

Comment:
Is this name defined by HTTP?
I thought that we just chose it for use out of a lot of other possible options (although it may be widely used).

Reply:
No, but cookie is a HTTP component. The "SID" is qbt defined name.
Do you disagree with moving the variable to here?

Reply:
>Do you disagree with moving the variable to here?

Yes.
I would keep this header/namespace for standart (protocol-defined etc.) names only.